### DISCUSSION TITLE
how can i download the docker images about milvus:v2.4.15

### DISCUSSION THREAD

**Discussion by huangxueliang** [milvus-io/milvus#37680] [Show and tell]
how can i download the docker images about milvus:v2.4.15? my windows not install docker destop
is there some tools to download?

**Comment by yhmo** [comment]
The official images are downloaded from the docker hub: https://hub.docker.com/r/milvusdb/milvus/tags
The standard command to pull a milvus image:
`docker pull milvusdb/milvus:v2.4.15`

If you cannot access it from China, you need to install a network proxy to bypass the firewall.

**Reply by yhmo** [reply]
@JackLCL might can help

**Comment by JackLCL** [comment]
@huangxueliang please add my wechat(13772075230), I will send you a docker image. BTW if you want to join milvus user wechat group, I can also invite you.
